Longest_Palindromic_Subsequence/Longest_Palindromic_Subsequence.java

```diff
@@ -0,0 +1,90 @@
+/*
+    LONGEST PALINDROMIC SUBSEQUENCE
+
+    The task is to find the length of the longest possible subsequence of a given word which is      a palindrome.
```

Comment:
Remove the extra spaces between **is** and **a**.

Divide the comment into more lines. Try to maintain a limit of 80 characters per line.

Do the same in C++ and python code.